qiskit/providers/basicaer/unitary_simulator.py

```diff
@@ -200,11 +200,8 @@ def _initialize_unitary(self):
                                    self._number_of_qubits * [2, 2])
 
     def _get_unitary(self):
-        """Return the current unitary in JSON Result spec format"""
+        """Return the current unitary"""
         unitary = np.reshape(self._unitary, 2 * [2 ** self._number_of_qubits])
-        # Expand complex numbers
-        unitary = np.stack((unitary.real, unitary.imag), axis=-1)
-        # Truncate small values
         unitary[abs(unitary) < self._chop_threshold] = 0.0
```

Comment:
I don't think this line is necessary if we return a numpy array, you can always just call `arr.round(decimals)` to truncate